hazelcast/src/main/java/com/hazelcast/config/PartitioningStrategyConfig.java

```diff
@@ -73,19 +73,25 @@ public PartitioningStrategyConfig setPartitioningStrategyClass(String partitionS
     }
 
     public PartitioningStrategy getPartitioningStrategy() {
-        return partitionStrategy;
+        return partitioningStrategy;
     }
 
+    @Deprecated
```

Comment:
"use setPartitioningStrategy() instead" ?

Reply:
yes, but since this is just a simple setter, I thought it was obvious.